rpa-home/modules/ROOT/pages/ms-automation-credits-usage-types.adoc

```diff
@@ -0,0 +1,40 @@
+= Automation Credits Usage and Rates
```

Comment:
I think we should try to focus on the JTBD instead of the feature. 
Said that I can not think of better titles than "Learning about...", "Getting to know...", etc.